content/people/damiri.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Delaram Amiri
+role: student-phd
+status: Ph.D. Candidate
+year joined: 2015
+portrait: damiri.jpg
+badge: damiri-badge.jpg
+interests:
+    - Healthcare IoT
+    - Sensor Control
+    - Optimization
+    - Edge Computing
+education:
+    - degree: M.S. in Electrical Engineering, Communications
+      institution: Indiana University - Purdue University Indianapolis, U.S.
```

Comment:
[Dash and spaces](https://github.com/uci-iasl/website/pull/10#discussion_r195909127).

Reply:
Not sure I did it correctly...
If not, how to get the en-dash?